test/guardianManager.js

```diff
@@ -324,6 +324,21 @@ describe("GuardianManager", function () {
       count = (await guardianStorage.guardianCount(wallet.contractAddress)).toNumber();
       assert.equal(count, 2, "there should be 2 guardians again");
     });
+
+    it("should be able to remove a guardian that is not last in the list", async () => {
```

Comment:
This says "_not_ last in the list" but the guardian revoked is `guardians[2]` which is the last in the list.

Reply:
Yes indeed I am testing this `if` condition logic https://github.com/argentlabs/argent-contracts/blob/develop/contracts/modules/storage/GuardianStorage.sol#L71 . Renamed.